src/main/groovy/wooga/gradle/appcenter/internal/DefaultAppCenterPluginExtension.groovy

```diff
@@ -17,92 +17,10 @@
 package wooga.gradle.appcenter.internal
 
 import org.gradle.api.Project
-import org.gradle.api.provider.ListProperty
-import org.gradle.api.provider.Property
 import wooga.gradle.appcenter.AppCenterPluginExtension
 
 class DefaultAppCenterPluginExtension implements AppCenterPluginExtension {
-    final Property<String> apiToken
-    final Property<String> owner
-    final Property<String> applicationIdentifier
-    final ListProperty<Map<String, String>> defaultDestinations
-    final Property<Boolean> publishEnabled
-    final Property<Long> retryTimeout
-    final Property<Integer> retryCount
 
     DefaultAppCenterPluginExtension(Project project) {
```

Comment:
you can update the constructor to no longer need the project reference. This is part of the internal API can be changed.

Reply:
Understood.